src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -334,6 +334,7 @@ module type S = sig
       ; fee_excess : Amount.Signed.t
       ; supply_increase : Amount.Signed.t
       ; protocol_state : Zkapp_precondition.Protocol_state.View.t
+      ; block_global_slot : Mina_numbers.Global_slot.t
```

Comment:
Nit: add a comment like `Slot of block when the transaction is applied. NOTE: This is at least 1 slot after the `protocol_state`'s view, which is for the *previous* slot.`